jfr-streaming/src/serialGcTest/java/io/opentelemetry/contrib/jfr/streaming/SerialGcMemoryMetricTest.java

```diff
@@ -48,27 +46,24 @@ void shouldHaveGCDurationMetrics() throws Exception {
                 .hasUnit(MILLISECONDS)
                 .hasDescription(METRIC_DESCRIPTION_GC_DURATION)
                 .satisfies(
-                    metricData -> {
-                      HistogramData data = metricData.getHistogramData();
-                      assertThat(data.getPoints())
-                          .map(HistogramPointData.class::cast)
-                          .anyMatch(
-                              p ->
-                                  p.getSum() > 0
-                                      && (p.getAttributes()
-                                              .equals(
-                                                  Attributes.of(
-                                                      ATTR_GC,
-                                                      "Copy",
-                                                      ATTR_ACTION,
-                                                      END_OF_MINOR_GC))
-                                          || p.getAttributes()
-                                              .equals(
-                                                  Attributes.of(
-                                                      ATTR_GC,
-                                                      "MarkSweepCompact",
-                                                      ATTR_ACTION,
-                                                      END_OF_MAJOR_GC))));
-                    }));
+                    data ->
+                        assertThat(data.getHistogramData().getPoints())
+                            .anyMatch(
+                                p ->
+                                    p.getSum() > 0
+                                        && (p.getAttributes()
+                                                .equals(
+                                                    Attributes.of(
+                                                        ATTR_GC,
+                                                        "Copy",
+                                                        ATTR_ACTION,
+                                                        END_OF_MINOR_GC))
+                                            || p.getAttributes()
+                                                .equals(
+                                                    Attributes.of(
+                                                        ATTR_GC,
+                                                        "MarkSweepCompact",
+                                                        ATTR_ACTION,
+                                                        END_OF_MAJOR_GC))))));
```

Comment:
Spotless makes this really hard to read! Maybe if those two attrs were local vars it could clean up a bit?

Reply:
Local vars improves it a lot 👍